### PR TITLE
Added backspace options for enable/disable backspace between words

### PR DESCRIPTION
**Pull request by gbrlmarn**
Hope this time is better 😀

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -55,6 +55,10 @@ struct Opt {
     /// List installed languages
     #[structopt(long)]
     list_languages: bool,
+
```
Comment (max-niederman) [inline src/main.rs:58]: Like I mentioned earlier, to introduce a change like this the default if no flags are specified would have to be the old behavior, or configurable (with the default config still being the old behavior).

The flag should be renamed to `--no-backtrack` or similar.

File: src/test/mod.rs [hunk 2]
```diff
@@ -52,7 +52,7 @@ impl Test {
         }
     }
 
-    pub fn handle_key(&mut self, key: KeyEvent) {
+    pub fn handle_key(&mut self, key: KeyEvent, backspace: bool) {
```
Comment (max-niederman) [inline src/test/mod.rs:55]: I would prefer to store the word backtracking configuration in the `Test` struct so it remains an (almost) pure state machine with keypresses as inputs.

**Review by max-niederman — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 1 above)
Comment (gbrlmarn) [reply src/main.rs:58]: I've done this change. Hope it's ok 😁

File: src/test/mod.rs (hunk 2 above)
Comment (gbrlmarn) [reply src/test/mod.rs]: Currently didn't know how to implement this and went with the old approach and with the other changes you suggested.

Comment (max-niederman) [reply src/test/mod.rs]: No worries, I can make the change.

**Review by max-niederman — APPROVED** [review]

**Pull request merged**